service/src/main/java/aero/minova/cas/MultipleLdapDomainsAuthenticationProvider.java

```diff
@@ -0,0 +1,66 @@
+package aero.minova.cas;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import org.springframework.security.authentication.AuthenticationProvider;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.AuthenticationException;
+import org.springframework.security.ldap.authentication.ad.ActiveDirectoryLdapAuthenticationProvider;
+import org.springframework.security.ldap.userdetails.UserDetailsContextMapper;
+
+public class MultipleLdapDomainsAuthenticationProvider implements AuthenticationProvider {
+
+	List<ActiveDirectoryLdapAuthenticationProvider> providers = new ArrayList<>();
+
+	public MultipleLdapDomainsAuthenticationProvider(String domains, String ldapServerAddresses, UserDetailsContextMapper userDetailsContextMapper) {
+
+		List<String> domainList = Arrays.asList(domains.split(SecurityConfig.MULTIPLE_LDAP_CONFIGURATIONS_SEPERATOR));
```

Comment:
Wenn die Konstante in SecurityConfig ist, würde ich das Split auch dort machen.
Die Konstante kann man somit auf private setzen und somit dieses Parsing in einer Klasse eingrenzen.